src/components/ClusterManager.js

```diff
@@ -306,11 +306,11 @@ export default class ClusterManager extends PureComponent {
         href: appLaunchLink,
         onClick: window.location.hash === appLaunchLink && currentStatus === 'Stopped' ? () => this.startCluster() : undefined,
         tooltip: canCompute ? `Open ${appName}` : noCompute,
-        'aria-label': `Open ${appName}`,
+        'aria-label': !isRStudioImage ? `Open ${appName}`: undefined,
```

Comment:
My vote would be to always include the label here, and give an empty alt attribute on the logo. Since this is primarily a control, it seems like we should describe the button directly. And the logo in this case doesn't add any additional information beyond that, so it seems fine to leave it empty.